modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordBinaryManager.kt

```diff
@@ -35,6 +36,7 @@ private const val DOWNLOAD_ENDPOINT = "https://github.com/metalbear-co/mirrord/r
 class MirrordBinaryManager {
     @Volatile
     private var latestSupportedVersion: String? = null
+    private var downloadVersion: String? = null
```

Comment:
Is `latestSupportedVersion` used at all now?

Reply:
both have been kept separate, as a user can specify a version like 3.60.0 and we were considering that as latestsupported version so yeah we need to keep both, but separate